src/components/ui/Tooltip.tsx

```diff
@@ -1,22 +1,30 @@
 import * as Popover from '@radix-ui/react-popover'
-import { ReactNode } from 'react'
+import { ReactNode, useState } from 'react'
 
 interface Props {
   content: string | ReactNode
   enabled?: boolean
   children: JSX.Element | JSX.Element [] | null
   className?: string
+  trigger?: 'click' | 'hover'
```

Comment:
I believe we default to click because hover isn't usable on web mobile

Reply:
i did set default to click below on line 16, but regardless you're right, hover won't work on mobile, so i can remove it.

Reply:
Let's not introduce a new behavior for Tooltip, and keep it consistent with existing UX (onClick show a tip, include a clickable external link if needed)

<img width="922" alt="tooltip" src="https://github.com/user-attachments/assets/b0a99729-ef02-43c4-ba5e-7e872542f947" />